docs/submitting-clinical-data.md

```diff
@@ -2,15 +2,28 @@
 id: submitting-clinical-data
 title: Submitting Clinical Data
 ---
+Clinical data consists of all supporting data about your programs donors and their health data.
 
-## Uploading Data
+> Note: Before you can submit clinical data, your data must be assigned ARGO Identifiers through the sample registration process. See [Registering Samples](/s) for instructions on how to register samples.
 
-### How To
 
-### Links
+## How To Submit Clinical Data
+1. Navigate to the **Submission** area in the top menu.
+1. Click on the **Clinical Submission** section in the left menu pane for your program.
+1. Download a **TSV Template** for the data type that you are submitting. Format your file(s) according to the [Data Dictionary](/dictionary) specifications for each clinical data type.
+1. Once your file(s) are formatted, click **Upload Files** and select the file. Only TSV file types are supported, and the file names must begin with data type that is being submitted.  For example, to _donor-\<anything\>.tsv_ would be a correctly formatted file name.
```

Comment:
...click the "Upload File" button in the top instructions box and select the file(s) from the file browser.

...would be a correctly formatted file name for your donor file.